herbert_core/classes/instrument_classes/instrument/detectors/@IX_detector_array/IX_detector_array.m

```diff
@@ -78,15 +84,32 @@
                         error('Detector indentifiers must all be unique')
                     end
                 else
-                    obj.det_bank_ = IX_detector_bank(varargin{:});
+                    [ok,mess] = IX_detector_array.check_detpar_parms(varargin{1});
```

Comment:
Seem to be ignoring the message. Do we want to keep returning `ok, mess` or use proper error handling?

Reply:
moving to proper error handling